examples/vercel-ai-chatbot/workspace/searchIndexes.ts

```diff
@@ -0,0 +1,6 @@
+import { DocumentIndexCreateRequest } from 'vellum-ai/api'
+
+export const cdktfIndex: Omit<DocumentIndexCreateRequest, 'indexingConfig'> = {
```

Comment:
Shows that a v0 could get pretty far with our API types. 

v1 is presumably our wrappers that may have `to_terraform()` and `execute()` methods.

v2 has us auto generating the resources based on your vellum api key and exposing them in the `vellum-ai//workspace` package directory